management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+from datetime import time
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
+        db = None
+        try:
+            while db is None:
+                try:
+                    db = self.check(databases=["default"])
```

Comment:
where is the implementation of `self.check`?

Reply:
where is the implementation of `self.check`?